tests/test_exception_handlers.py

```diff
@@ -65,3 +77,12 @@ def test_override_server_error_exception_response():
     response = client.get("/server-error")
     assert response.status_code == 500
     assert response.json() == {"exception": "server-error"}
+
+
+def test_traceback_for_dependency_with_yield():
+    client = TestClient(app, raise_server_exceptions=True)
+    with pytest.raises(ValueError) as exc_info:
+        client.get("/dependency-with-yield")
+    last_frame = exc_info.traceback[-1]
+    assert str(last_frame.path) == __file__
+    assert last_frame.lineno == raise_value_error.__code__.co_firstlineno
```

Comment:
If you revert the changes, you can verify that the dependency will not be the last frame (or in any frame, because it will not be present).

Reply:
Awesome, great, I did this to verify. :nerd_face: